core/tests/test_utils.py

```diff
@@ -930,6 +999,71 @@ def save_new_exp_with_states_schema_v0(self, exp_id, user_id, title):
         exploration = exp_services.get_exploration_from_model(exp_model)
         exp_services.create_and_save_state_id_mapping_model(exploration, [])
 
+    def save_new_exp_with_states_schema_v21(self, exp_id, user_id, title):
```

Comment:
Can you explain in the docstring the significance of version 21? Just a note at the end of the first paragraph saying "Version 21 is where XXX is implemented." will do.